sync/lib/fetch.go

```diff
@@ -64,6 +65,21 @@ func (s *LocalFetch) GetFile(file *pki.PKIFile) (*pki.SeekFile, error) {
         s.Log.Debugf("Fetching %v->%v", file.Path, newPath)
     }
     data, err := FetchFile(newPath, file.Type != pki.TYPE_TAL)
+    if err != nil {
+        var contained bool
+        for _, pa := range s.PathAvailable {
+            contained = strings.Contains(file.Path, pa)
+            if contained {
+                break
+            }
+        }
+        if !contained {
+            if s.Log != nil {
+                s.Log.Infof("Error on repository not yet synchronized: %v", err)
```

Comment:
Why use the string "Error" here when you're only raising a log with priority "Info"?

Reply:
Fixed -> `Got %v but repository not yet synchronized`